grove/alpha/fermion_transforms/tests/test_bravyi_kitaev.py

```diff
@@ -16,8 +16,8 @@ def test_hardcoded_transform():
     x = bkt.kill(9)
     y = bkt.create(9)
 
-    assert str(x) == '0.5j*Z7*Y9*X11*X15 + (0.5+0j)*Z7*Z8*X9*X11*X15'
-    assert str(y) == '-0.5j*Z7*Y9*X11*X15 + (0.5+0j)*Z7*Z8*X9*X11*X15'
+    assert str(x) == '0.5j*Z7*Y9*X11*X15 + 0.5*Z7*Z8*X9*X11*X15'
```

Comment:
those edits are unrelated to grover, please leave them out and pout them in a separate PR

Reply:
I think my rebase was only partial - these are reverting the good change. A rebase should fix this.